backend/src/ee/services/access-approval-request/access-approval-request-service.ts

```diff
@@ -130,6 +130,9 @@ export const accessApprovalRequestServiceFactory = ({
         message: `No policy in environment with slug '${environment.slug}' and with secret path '${secretPath}' was found.`
       });
     }
+    if (policy.deletedAt) {
+      throw new BadRequestError({ message: "The policy linked to this request has been deleted" });
```

Comment:
Error would make more sense if it was `You can't create access requests for a policy that has been deleted`

Reply:
"You can't create access requests for a policy that has been deleted"
this makes it seem like they already know (or should know) the policy has been deleted

"The policy linked to this request has been deleted"
this informs them that the policy has been deleted, and since an error will show, they would assume its why they cant create a new request/approval

2 different approaches